tpkg/ede.tdir/ede.pre

```diff
@@ -0,0 +1,21 @@
+# #-- ede.pre--#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+. ../common.sh
+
+# start NSD
+get_random_port 1
+TPKG_PORT=$RND_PORT
+
+PRE="../.."
+TPKG_NSD_PID="$PRE/nsd.pid.$$"
```

Comment:
It is better to have the test files inside the test directory, so not write the pidfile outside of it with `$PRE` in front.  Even though with the `$$` it is uniquified, I suggest to remove the `$PRE` to put the pidfile inside the test directory, because the current directory is a temporary copy.

```suggestion
TPKG_NSD_PID="nsd.pid.$$"
```

Reply:
ack! We copied nsid_ascii.tdir, and this came with it. We will fix!